docs/openapi/components/schemas/credentials/GS1DelegationCredential.yml

```diff
@@ -0,0 +1,120 @@
+$linkedData:
+  term: GS1DelegationCredential
+  '@id': https://w3id.org/traceability#GS1DelegationCredential
+title: GS1 Delegation Credential
+tags:
+  - eCommerce
+description: >-
+  This is the Verifiable Credential that delegates to another party the right to issue data credentials on behalf of the issuer of this credential.
+type: object
+properties:
+  '@context':
+    type: array
+    readOnly: true
+    const:
+      - https://www.w3.org/2018/credentials/v1
+      - https://w3id.org/traceability/v1
+      - https://w3id.org/vc/status-list/2021/v1
```

Comment:
Similar revocation conversation.

Reply:
Yep - same answer: I suggest we let this though to avoid a dependency gridlock.

Reply:
beware that we might need to "update tooling" every-time we "add a new context"....